vcnc-server/vcnc-sampler/app.js

```diff
@@ -87,31 +84,51 @@ function requestHandler(request, response) {
   response.end();
 }
 
-const server = http.createServer(requestHandler);
-server.setTimeout(5000, () => {
-});
+// Create server
+//
+let server;
+function startServer(h, p) {
+  server = http.createServer(requestHandler);
+  server.setTimeout(5000, () => {
+  });
```

Comment:
I didn't want to hold channel for a long time because vda could send messages to vcnc, elasticsearch, etc.